test/unit/math/fwd/mat/vectorize/apply_scalar_unary_test.cpp

```diff
@@ -0,0 +1,17 @@
+#include <stan/math/fwd/core/fvar.hpp>
```

Comment:
replace this block with:
```
#include <stan/math/fwd/mat.hpp>
#include <gtest/gtest.h>
#include <stan/math/fwd/mat/vectorize/apply_scalar_unary.hpp>
#include <test/unit/math/prim/mat/vectorize/expect_types.hpp>
#include <test/unit/math/fwd/mat/vectorize/expect_values.hpp>
#include <test/unit/math/fwd/mat/vectorize/expect_errors.hpp>
#include <test/unit/math/prim/mat/vectorize/foo_base_test.hpp>
```

This is due to the a new pull request that got into develop. It'll make includes easier.